deployment/eks/flyte_generated.yaml

```diff
@@ -7610,6 +7610,12 @@ metadata:
 ---
 apiVersion: v1
 kind: ServiceAccount
+metadata:
```

Comment:
cc @anandswaminathan this should not impact Lyft, but this is a new dependency, that if you want to use `secrets` in flytekit declaration then you have to enable

Reply:
ohhh